model_compression_toolkit/core/common/graph/functional_node.py

```diff
@@ -71,3 +72,17 @@ def type(self):
         :return: the node's functional_op
         """
         return self.functional_op
+
+    def is_match_type(self, _type: Type) -> bool:
+        """
+        Check if input type matches the node type, either in instance type or in type name. Checking the
+        name string is required because of function types changes that occurred in TF 2.15.
+
+        Args:
+            _type: other node type
+        Returns:
+            Whether _type matches the self node type
+
+        """
+        names_match = _type.__name__ == self.type.__name__ if FOUND_TF else False
```

Comment:
Very nice!
I just think that it needs more explaining (besides the one sentence mentioning tf 2.15 in the function's documentation)